main/src/vtr_radar/src/modules/preprocessing/conversions/offline_radar_conversion_module.cpp

```diff
@@ -142,6 +142,9 @@ void OfflineRadarConversionModule::run_(QueryCache &qdata0, OutputCache &,
   qdata.radar_data->cartesian = cartesian;
   qdata.radar_data->azimuth_times = azimuth_times;
   qdata.radar_data->azimuth_angles = azimuth_angles;  
+
+  /// make a non-null pointer here to indicate that the radar data has been loaded
+  qdata.scan_msg.emplace();
```

Comment:
I don't like this. Why is this nessecary?

Reply:
it is checked everywhere cuz offline does not have this message and gyro preintegration and odometry and localization checks this data and relies on this data being set. To make Daniil pipeline work as intended I think it runs now